assets/app/styles/_messages.less

```diff
@@ -160,27 +164,20 @@ ul.messenger-theme-flat .messenger-message .messenger-actions a {
 ul.messenger-theme-flat {
   .messenger-message.alert-error {
     &:before {
-      content: "\e609";
+      content: "\e61c";
```

Comment:
@jwforres Good suggestion.  In this case, you'd still have to have changed the variable in all these places since the icon itself was changing, no?